libMBIN/Source/NMS/Toolkit/TkMaterialMetaData.cs

```diff
@@ -84,26 +84,31 @@ public enum DecalNormalEnum { None, Decal, Vertex }
         /* 0x390 */ public float ShellsHeight;
         /* 0x394 */ public bool Dissolve;
         /* 0x395 */ public bool InvertAlpha;
-        /* 0x396 */ public bool StretchyParticle;
-        /* 0x397 */ public bool UVScrolling;
-        /* 0x398 */ public float UVScrollX;
-        /* 0x39C */ public float UVScrollY;
-        /* 0x3A0 */ public float UVScrollNormalX;
-        /* 0x3A4 */ public float UVScrollNormalY;
-        /* 0x3A8 */ public float UVFrameTime;
-        /* 0x3AC */ public bool UVAnimation;
-        /* 0x3AD */ public bool UVTileAlts;
-        /* 0x3B0 */ public float UVNumTilesX;
-        /* 0x3B4 */ public float UVNumTilesY;
-        /* 0x3B8 */ public bool DisplacementWave;
-        /* 0x3BC */ public float WaveOneSpeed;
-        /* 0x3C0 */ public Vector4f WaveOneAmplitude;
-        /* 0x3D0 */ public Vector4f WaveOneFrequency;
-        /* 0x3E0 */ public Vector4f WaveOneFallOff;
-        /* 0x3F0 */ public float WaveTwoSpeed;
-        /* 0x400 */ public Vector4f WaveTwoAmplitude;
-        /* 0x410 */ public Vector4f WaveTwoFrequency;
-        /* 0x420 */ public Vector4f WaveTwoFallOff;
+        /* 0x396 */ public bool BeforeUI;
+        /* 0x397 */ public bool DepthMaskUI;
+        /* 0x398 */ public bool AlwaysOnTopUI;
+        /* 0x399 */ public bool StretchyParticle;
+        /* 0x39A */ public bool UVScrolling;
+        [NMS(Size = 0x1, Ignore = true)]
```

Comment:
technically not needed but doesn't matter :P

Reply:
Can we leave it there?  I like explicit things and it shows that 0x39B is not an error but intended...

Reply:
I mean this line and the one below (ie. no padding is needed since the float will be aligned to 0x4 anyway...) But as I said, it doesn't matter :)

Reply:
You mean the next field after the bool at 0x39A being a float will cause it to go to 0x39C anyway without the padding?

Reply:
exactly!